docs/api/frameless-window.md

```diff
@@ -18,25 +18,27 @@ const win = new BrowserWindow({ width: 800, height: 600, frame: false })
 win.show()
 ```
 
-### Alternatives on macOS
+### Alternatives
 
-There's an alternative way to specify a chromeless window.
+There's an alternative way to specify a chromeless window on macOS and Windows.
 Instead of setting `frame` to `false` which disables both the titlebar and window controls,
 you may want to have the title bar hidden and your content extend to the full window size,
-yet still preserve the window controls ("traffic lights") for standard window actions.
+yet still preserve the window controls ("traffic lights" on macOS) for standard window actions.
 You can do so by specifying the `titleBarStyle` option:
 
-#### `hidden`
+### `hidden`
```

Comment:
I'm not sure why these headings got changed. I think it looks better like it was originally.
```suggestion
#### `hidden`
```

Reply:
I believe these are just remnants of when `hidden` was going to be used to turn the feature on. I agree the original looks better.